test/parallel/test-util-callbackify.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const common = require('../common');
+
+
+const assert = require('assert');
+const { callbackify } = require('util');
+
+
+{
+  const sentinel = 'hello world';
+  async function fn() {
+    return await Promise.resolve(sentinel);
+  }
+  const cbFn = callbackify(fn);
+
+  cbFn(common.mustCall((err, ret) => {
+    if (err) assert.ifError(err);
```

Comment:
I think using `if (err)` here is redundant because `assert.ifError` already is a no-op if `err` is falsy.